tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -5303,23 +5303,20 @@ public function findEntryFromPropfindResponse(
 			default:
 				throw new Exception("error");
 		}
+		//while doing report request, all dav path will give spaces dev spaces end point in response
```

Comment:
could you re write the comment hard to understand 
```
		//while doing report request, all dav path will give spaces dev spaces end point in response
```

Reply:
`All dav paths will respond with a spaces dev spaces end point when a report request is made.`
Is this will be ok ??